webpack-test/statsCases/performance-different-mode-and-target/webpack.config.js

```diff
@@ -14,6 +17,9 @@ module.exports = [
 		target: "webworker",
 		output: {
 			filename: "warning.pro-webworker.js"
+		},
+		stats: {
+			hash: false,
```

Comment:
If hash is not set to false, what problems will occur?

Reply:
I don't know why either. When running `npm run test:webpack`, the generated hashes are different locally and on the CI. https://github.com/web-infra-dev/rspack/actions/runs/8980383687/job/24664828197

Reply:
Caused by hash generating changed on main branch.